karma.conf.js

```diff
@@ -151,20 +177,43 @@ module.exports = function(config) {
 				// rather than referencing source files inside the module
 				// directly
 				alias: {
-					'preact/hooks': path.join(__dirname, './hooks/src'),
```

Comment:
If we just change these aliases to the directory that contains the package.json, does webpack pick up our build output? That way we don't have to minify again in the karma-webpack and are testing the same build output our users will consume.

Reply:
It'll break watch mode, but otherwise I think this might be good - it will account for microbundle issues if they exist.

Reply:
Need to push my latest changes. I have it close to working except an issue with our schedule bundle 👍